### DISCUSSION TITLE
Flow not saving logs

### DISCUSSION THREAD

**Discussion by Swizzle4372** [jongpie/NebulaLogger#699] [Q&A] [answered]
Hi.  I'm trying to create logs from flows and having no luck in either sandboxes or production.  Take this simple flow:
![image](https://github.com/jongpie/NebulaLogger/assets/19466487/6b39e0c5-f53f-4f45-9fbf-b2b31d6d41e7)

I've set Save Log to true and still I'm getting no logs created.  I tried having that set to false and using the separate Save Log action but still no logs.  The debug log shows no errors and debugging the flow shows that the action returns a transaction Id:
![image](https://github.com/jongpie/NebulaLogger/assets/19466487/6ce9ee62-0a6d-417d-857c-53134296976a)

I have the Nebula Logger: Admin and Nebula Logger: Log Creator permission sets assigned to myself.

Logs are being created through Apex and LWC successfully, this is just a flow specific issue.
I've updated to 4.13.11 but that hasn't fixed it.

I read https://github.com/jongpie/NebulaLogger/discussions/675 but that seems to be caused by user error (I actually just added a comment on that issue to point that out).

Any suggestions?  Do I need to enable some other permission or setting in order to create logs through flows?  Thanks

**Comment by jongpie** [comment, marked as answer]
@ministe2003 very strange! Offhand, I don't see any issues with what you've setup in Flow, but there could be a few other things that might be causing the problem:

- Validate that you have `DEBUG` logging level enabled via `LoggerSettings__c` - since you're not setting the "(Optional) Logging Level" property in your Flow, it uses `DEBUG` by default. If you don't have `DEBUG` enabled via `LoggerSettings__c.LoggingLevel__c`, then it won't save the log entry
- You can also try updating your Flow to set the property "(Optional) Save Method" to be `SYNCHRONOUS_DML` - this save method will immediately try to create records in `Log__c` & `LogEntry__c`, and can help surface hidden errors in Nebula Logger

Let me know if either of those options helps in your org!

**Reply by Swizzle4372** [reply]
Ah you've cracked it, I thought it'd be something simple!  Our logging level was set to ERROR, so once I added that to the logging level field, the log is being stored.  As you can probably tell, we're new to Nebula!

Thanks very much!

**Reply by jongpie** [reply]
@ministe2003 that's great to hear! Glad you were able to get everything working. If you need anything else, just let me know!
